presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -59,14 +66,28 @@ public void testReadFullAcid(boolean isPartitioned, BucketingType bucketingType)
             // test filtering
             assertThat(query("SELECT col, fcol FROM " + tableName + " WHERE fcol = 1 ORDER BY col")).containsOnly(row(21, 1));
 
+            // test compacted data read
+            onHive().executeQuery(
+                    "INSERT INTO TABLE " + tableName + hivePartitionString + " VALUES (20, 3)");
+            ExecutorService executor = Executors.newSingleThreadExecutor();
+            try {
+                Future result = executor.submit(() ->
+                        onHive().executeQuery("alter table " + tableName + " " + hivePartitionString + " compact 'minor' and wait"));
```

Comment:
upper case SQL keywords (and 'minor' too)

Reply:
should we test with MAJOR compaction too?

Reply:
will do.

I didnt add major compaction tests as they were not in the context of this fix. Should I add them in the same commit or a separate one?

Reply:
can be in one

Reply:
i would consider adding an additional enum parameter to test methods like 
```
enum Compaction { NONE, MINOR, MAJOR }
```

so that the test code remains shorter, not "an epic story" testing various cases one by one

Reply:
That would increase the number of tests three times and each of these tests is time taking so overall increase in test suite runtime will be considerable. Is that okay @findepi ?

Reply:
Thanks for raising this up. How many seconds will this add?

Reply:
In current PR runs (https://github.com/prestosql/presto/pull/3437/checks?check_run_id=588238426), 16 tests of testReadFullAcid took 8minutes. It would almost triple.

Reply:
Handled casing but left out Compaction as new test parameter, instead added major and minor compaction tests in method